node/test/performance-test/build.rs

```diff
@@ -0,0 +1,21 @@
+// Copyright 2015-2020 Parity Technologies (UK) Ltd.
```

Comment:
```suggestion
// Copyright 2021 Parity Technologies (UK) Ltd.
```

Reply:
Oops, a copy-paste of the cli's build.rs